Wikipedia/Code/WMFArticleViewController.m

```diff
@@ -189,7 +189,7 @@ - (instancetype)initWithArticleURL:(NSURL *)url
     if (self) {
         NSString *fragment = [url fragment];
         if (![[fragment stringByTrimmingCharactersInSet:[NSCharacterSet whitespaceAndNewlineCharacterSet]] isEqualToString:@""]) {
-            self.initialFragment = [fragment stringByRemovingPercentEncoding];
+            self.initialAnchor = [fragment stringByRemovingPercentEncoding];
```

Comment:
`fragment` makes sense in the context of a URL, for our use case I renamed everything to anchor to better correspond with [wiki naming conventions](https://meta.wikimedia.org/wiki/Help:Link#Anchors) and `MWKSection`.